airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGeneratorIntegrationTest.java

```diff
@@ -708,7 +726,7 @@ -- insert raw record from the second record batch - this is an outdated record t
     final long finalRows = bq.query(QueryJobConfiguration.newBuilder("SELECT * FROM " + streamId.finalTableId("", QUOTE)).build()).getTotalRows();
     assertEquals(0, finalRows);
     final long rawRows = bq.query(QueryJobConfiguration.newBuilder("SELECT * FROM " + streamId.rawTableId(QUOTE)).build()).getTotalRows();
-    assertEquals(1, rawRows);
+    assertEquals(2, rawRows); // we keep the old and the new raw record in this out-of-order case
```

Comment:
This is a logical change I needed to make, to keep the latest + the raw record with the delete present. I think that's fine 🤷

Reply:
:+1: feels fine since it's in the raw table :shrug: